quupe-RN/js/screens/TransactionHistory/TransactionHistory.js

```diff
@@ -1,11 +1,109 @@
 import React, { Component } from 'react';
-import { Text, View } from 'react-native';
-
-const TransactionHistory = () => {
-    return (
-        <View>
-            <Text> TransactionHistory </Text>
-        </View>
-    );
+import { Text, View, TouchableOpacity, ScrollView, Image } from 'react-native';
+import Icon from 'react-native-vector-icons/Ionicons';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+
+class TransactionHistory extends Component {
+    constructor() {
+        super();
+        this.state = {
```

Comment:
We do not use component states, please use redux or come and talk to me we can find something better.